metals/src/main/scala/scala/meta/internal/metals/debug/DebugProvider.scala

```diff
@@ -440,6 +443,10 @@ class DebugProvider(
       languageClient.showMessage(
         Messages.errorMessageParams(e.getMessage())
       )
+    case _: ju.NoSuchElementException =>
```

Comment:
Could we add ` BuildTargetUndefined` exception just in case? and throw that in 407?

Reply:
Sure! I'll commit this right away, and you're right about the issue, I fixed it! :D